README.md

```diff
@@ -1,2 +1,42 @@
-# LeaderBoard
-Leaderboard applicaton | JavaScript
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Leaderboard
+
+> Leader board  application using Js/webpack
```

Comment:
- Please, kindly add more description about your project to give users an insight of what the project i all about